test/README.md

```diff
@@ -444,6 +444,26 @@ All of the functions that act as the scenario API are run in the
 context of `./test/bin/scenario.sh` and can therefore use any functions
 defined there.
 
+#### Scenarios coverage
+
+Scenarios utilize following distinct MicroShift sources:
+- `src`: built from source (code in PR)
+- `base`: built from base branch (PR's target branch)
+- `prev`: previous MicroShift minor release
```

Comment:
Just a thought: `rel` or `release` might imply more clearly what the source is.

Reply:
Yeah, `prev` might not be most descriptive in this case, but it originates from `prev-minor` and it's meant to point to a previous Z stream, so for `4.14` it's `4.13`.

In the readme change I mentioned `released`/`rel` which I want to use together with ECs, RCs, and finally Z-stream releases of $current release, like `4.14~ec.N`, etc.

If we can come up with something short and sweet then we can change it :) But for now, we're facing one of most difficult challenges - naming things

Reply:
What about `prel`, `crel`, etc.?

Reply:
Sure, we can agree on these for now and if we'll discover some new reqs in 4.15, we'll rethink